### ISSUE TITLE
Alexa option disappeared from drop down menu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#116]
```
What steps will reproduce the problem?
1. Launch Interface

What is the expected output? What do you see instead?
The standard selection menu with all options, but the Alexa choice isn't
showing up.

What version of the product are you using? On what operating system?
Running 1.2 on Windows 7 Ultimate 64-bit

Please provide any additional information below.
I have run this before with the option coming up correctly, but it seems to
have mysteriously disappeared, I have cleared my temp and re-downloaded the
program already. I can still find the alexa domain list file in the temp
when it's running. Perhaps it would be nice to have the option to add any
text file you'd like for testing, and have the alexa list just be one
that's included.

```

Original issue reported on code.google.com by `mccl...@gmail.com` on 1 May 2010 at 1:26

**Comment by GoogleCodeExporter** [comment]
```
The Alexa Top 10,000 list is old. I looked through one of the log files in the
Windows temp folder, and found that error, which said it was being ignored.

"2010-05-01 22:20:23.255000: Ignoring data\alexa-top-10000-global.txt from 
alexa (64
days old)"

As a temporary workaround, you can simply open up the 
alexa-top-10000-global.txt and
make a small edit, save, undo the edit, and save again. This will chance the 
save
date to today, which will allow the program to use that file.

The alexa-top-10000-global.txt should be located under %temp%\namebench\data\.


Hope this helps!
```

Original comment by `thevp...@gmail.com` on 1 May 2010 at 3:28

**Comment by GoogleCodeExporter** [comment]
```
Thanks for debugging this - it has been fixed in r757, and will be part of a 
new release early next week.
```

Original comment by `helixblue` on 1 May 2010 at 6:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Ah, this is good news, I was planning on making a "tutoreview" on this, but I 
had to
stop when the feature vanished. I guess I'll wait for the new release and make 
it then.
```

Original comment by `mccl...@gmail.com` on 9 May 2010 at 1:32

**Comment by GoogleCodeExporter** [comment]
```
Can you try the newly released namebench 1.3-BETA1 to see if it resolves this 
bug?
```

Original comment by `helixblue` on 20 May 2010 at 3:21

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3.

```

Original comment by `helixblue` on 31 May 2010 at 6:44
- Changed state: **Fixed**
